solutions/haskell/04-positive_character_groups/definition.yml

```diff
@@ -0,0 +1,21 @@
+author_details:
+  - name: Farhad Mehta
```

Comment:
If it's okay with Farhad, can we remove him from authors and just keep in reviewers? I think it'd be confusing to users to see the same person be the author and reviewer. 

I see that Benjamin's PR uses the format suggested above: 

<img width="873" alt="Screenshot 2022-12-09 at 10 22 05" src="https://user-images.githubusercontent.com/3893573/206680466-06a551bf-4b21-473d-8019-6dc69ce4023e.png">

Reply:
Done.